addons/frag/functions/fnc_dev_trackHitBox.sqf

```diff
@@ -0,0 +1,73 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add a hitbox outline to an object
+ *
+ * Arguments:
+ * 0 Object to draw hitbox <OBJECT>:
+ * 1: Add sphere at object origin <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call ace_frag_fnc_dev_trackHitBox;
+ *
+ * Public: No
+ */
+params [
+    ["_object", objNull, [objNull]],
+    ["_addSphere", true, [true]]
+];
+
+if (isNull _object) exitWith {};
+
+// Grab the right hitBox
+private _box = [];
+if (_object  isKindOf "CAManBase") then {
```

Comment:
```suggestion
if (_object isKindOf "CAManBase") then {
```

Reply:
Went through and found every instance of undesired repeat spaces and removed them here https://github.com/acemod/ACE3/pull/9728/commits/054b9cda2367f481e66677af914870008237adcf